.travis.yml

```diff
@@ -56,19 +67,21 @@ install:
       exit 1;
     fi
 
-  # Restore dependencies
-  - dotnet restore
+  # Hack for: https://github.com/dotnet/cli/issues/3658
+  - if test "$TRAVIS_OS_NAME" == "linux"; then
+      nvm install stable && nvm use stable;
+    fi
```

Comment:
This is not required. That bug only affects `dotnet run` command (which we were using for Benchmark project in GeoIP2); the unit test project now runs with `dotnet test` command.